Resources/public/js/views/dashboard/ez-dashboardblockallcontentview.js

```diff
@@ -0,0 +1,228 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-dashboardblockallcontentview', function (Y) {
+    'use strict';
+
+    /**
+     * Provides the All Content Dashboard Block View class
+     *
+     * @module ez-dashboardblockallcontentview
+     */
+    Y.namespace('eZ');
+
+    var SELECTOR_CONTENT = '.ez-allcontent-block-content';
+
+    /**
+     * The all content dashboard block view
+     *
+     * @namespace eZ
+     * @class DashboardBlockAllContentView
+     * @constructor
+     * @extends eZ.DashboardBlockBaseView
+     */
+    Y.eZ.DashboardBlockAllContentView = Y.Base.create('dashboardBlockAllContentView', Y.eZ.DashboardBlockBaseView, [Y.eZ.AsynchronousView], {
+        initializer: function () {
+            var parentView = this.constructor.superclass;
+
+            this._fireMethod = this._getContentData;
+            this._watchAttribute = 'contentList';
+
+            this.get('container').addClass(parentView._generateViewClassName(parentView._getName()));
+        },
+
+        /**
+         * Renders the dashboard block view
+         *
+         * @method render
+         * @return {eZ.DashboardBlockView} the view itself
+         */
+        render: function () {
+            this.get('container').setHTML(this.template(this.getAttrs()));
```

Comment:
Why? I don't see any difference when specifying template variables here. I would say, that my approach is simpler and less complicated.